bin/pygrb/pycbc_grb_inj_finder

```diff
@@ -329,12 +337,21 @@ with h5py.File(outfilename, "w") as h5out:
     for x in out_trigs:
         xgroup = h5out.create_group(x)
         for col in out_trigs[x]:
+            if "template_id" in col:
```

Comment:
Should this not be `if col == "template_id":`? Or does `col` contain other characters in addition?

Reply:
`col` is the full column header, so it includes the IFO/network tags.

Reply:
Are you sure? In the code above I see
```python
for x in triggers:
    ij = x.split('/')
    out_trigs[ij[0]][ij[1]].append(triggers[x][eid])
```
so it seems to me that `out_trigs` is a dict-of-dict type object, where the outer key is the interferometer or "network", and the inner key is the field name.

Reply:
Oh right, I was thinking about `pycbc_grb_trig_combiner`. You're correct!